diff -aurN tutka-0.12.3.orig/tutka.desktop tutka-0.12.3/tutka.desktop
--- tutka-0.12.3.orig/tutka.desktop	2005-04-03 23:44:29.000000000 +0200
+++ tutka-0.12.3/tutka.desktop	2005-04-23 16:45:39.000000000 +0200
@@ -1,9 +1,11 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=Tutka MIDI Tracker
-Comment=Compose music using MIDI
+Name=Tutka
+Comment=Tracker style MIDI sequencer
+Comment[pl]=Sekwencer MIDI w stylu trackera
 Exec=tutka
 Icon=tutka
 Terminal=false
 Type=Application
-Categories=GNOME;Application;AudioVideo;Audio;Midi;
+Categories=GTK;GNOME;Audio;Sequencer;
+# vi: encoding=utf-8
