diff -aurN tutka-0.11.2.orig/tutka.desktop tutka-0.11.2/tutka.desktop
--- tutka-0.11.2.orig/tutka.desktop	2003-01-22 18:15:06.000000000 +0100
+++ tutka-0.11.2/tutka.desktop	2004-08-18 11:28:06.691155816 +0200
@@ -1,9 +1,11 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=Tutka MIDI Tracker
-Comment=Compose music using MIDI
+Name=Tutka
+Comment=Tracker style MIDI sequencer
+Comment[pl]=Sekwencer MIDI w stylu trackera
 Exec=tutka
 Icon=tutka.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;AudioVideo;
+Categories=GTK;GNOME;Audio;Sequencer;
+# vi: encoding=utf-8
