diff -aurN tutka-0.12.3.orig/Makefile.am tutka-0.12.3/Makefile.am
--- tutka-0.12.3.orig/Makefile.am	2005-04-03 22:28:37.000000000 +0200
+++ tutka-0.12.3/Makefile.am	2005-04-23 16:48:06.000000000 +0200
@@ -26,7 +26,6 @@
 	$(INSTALL_DATA) tutka.png $(DESTDIR)$(datadir)/pixmaps
 	$(mkinstalldirs) $(DESTDIR)$(GCONF_SCHEMA_FILE_DIR)
 	$(INSTALL_DATA) tutka.schemas $(DESTDIR)$(GCONF_SCHEMA_FILE_DIR)
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule tutka.schemas
 
 dist-hook:
 	if test -d pixmaps; then \
